lib/beaker/host/unix/exec.rb

```diff
@@ -246,7 +246,7 @@ def environment_variable_string_pair_array env
       else
         val = val.to_s
       end
-      env_array << "#{key.to_s.upcase}=\"#{val}\""
```

Comment:
This has the likely potential to break existing tests that are relying on the old beaker behavior. If we are going to release this in beaker 3.x, then we need to preserve the old behavior as well (so set both the `upcase` and original values). On Windows, they env variables will overwrite each other, with the same value, so not an issue. On *nix, there will be two env variables, but they will have the same value, so again no issue.